frontend/src/scenes/insights/EditorFilters/EditorFilterGroup.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { InsightEditorFilterGroup, InsightLogicProps, InsightModel } from '~/types'
+import { cleanFilters } from '../utils/cleanFilters'
+import { EditorFilterItemTitle } from './EditorFilterItemTitle'
+import './EditorFilterGroup.scss'
+import { LemonButton } from 'lib/components/LemonButton'
+import { IconUnfoldLess, IconUnfoldMore } from 'lib/components/icons'
+
+export interface EditorFilterGroupProps {
+    editorFilterGroup: InsightEditorFilterGroup
+    insight: Partial<InsightModel>
+    insightProps: InsightLogicProps
+}
+
+export function EditorFilterGroup({ editorFilterGroup, insight, insightProps }: EditorFilterGroupProps): JSX.Element {
+    const { title, editorFilters, defaultExpanded } = editorFilterGroup
+    const [isRowExpanded, setIsRowExpanded] = useState(defaultExpanded)
```

Comment:
Scope creep alert (please defer): I wonder if we should also store the expanded/collapsed state in a filter, and save it with the insight? Perhaps not?

Reply:
I thought about this too but I think the potential branches are weird. I would vote we play with it and see what feels rights, gather some data and make a decision then